Governance/concepts/other-conventions.md

```diff
@@ -17,14 +17,15 @@ IMOS] has very specific extensions of the CF conventions for different kind of o
 * [OceanSITES Data Format](http://www.oceansites.org/docs/oceansites_data_format_reference_manual.pdf) - extension of CF Conventions standard for OceanSITES data 
 * [ARGO netCDF conventions for data centers](http://www.argodatamgt.org/Media/Medias-Argo-Data-Management/Argo-documentation/General-documentation/Data-format)
 * [PMEL-EPIC Conventions](https://www.pmel.noaa.gov/epic/document/convention.htm)
-* [US NCEI NetCDF templates](https://www.ncei.noaa.gov/netcdf-templates) - these templates illustrate how to apply CF and ACDD conventions
+* [US NCEI netCDF templates](https://www.ncei.noaa.gov/netcdf-templates) - these templates illustrate how to apply CF and ACDD conventions
 
 ## Atmosphere 
 * [NCAR-RAF Conventions for Aircraft Data](http://www.eol.ucar.edu/raf/Software/netCDF.html)
 
 ## Other 
 * [AMBER Trajectory Conventions](http://ambermd.org/netcdf/nctraj.xhtml) for molecular dynamics simulations.
 * [CF Discrete Sampling Geometries Conventions](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.6/build/cf-conventions.html\#discrete-sampling-geometries) - CF for observational and point data 
+* [COMODO](https://web.archive.org/web/20160417032300/http://pycomodo.forge.imag.fr/norm.html) ??? still used?
 * [UGRID Conventions](http://ugrid-conventions.github.io/ugrid-conventions/) for unstructured (e.g. triangular, hex) grids. 
```

Comment:
I don't know if it's just the modelling world I live in but I'd be tempted to move this to the top of the list, as the number of unstructured grid models we interact with grows (and people make more use of unstructured grids in existing models). It is becoming increasingly important (for me) for people to be making their data UGRID compliant as well for tools to be able to read it.

Reply:
you mean grid and commode? I was wondering about the second as it's completely offline, NB the link is an archived page, but it looked important

Reply:
Whoops no I mis-clicked, just UGRID. 
I don't know anything about COMODO!

Reply:
it's a package to define Arakawa-C grids, but it looks like abandoned, I'll leave it there until we've worked that one out and I'll promote UGRID, right now the list was just alphabetical